observer/package.json

```diff
@@ -3,18 +3,20 @@
   "type": "module",
   "private": true,
   "scripts": {
+    "dry-run": "node bin/dry-run.js",
     "migrate": "node bin/migrate.js",
     "start": "node bin/spark-observer.js",
     "lint": "standard",
     "test": "mocha"
   },
   "devDependencies": {
     "mocha": "^10.4.0",
+    "spark-evaluate": "github:filecoin-station/spark-evaluate#534454c9b2323b67d50b159a3504de8a06eedcb2",
```

Comment:
I think we should use `#main` in the package json file. Unless you are certain that Dependabot will bump up the commit sha for us.